.github/workflows/docker-build-tag.yml

```diff
@@ -111,13 +112,14 @@ jobs:
             cd ./scripts
             ./chart_releaser.sh --helm-chart-folder testkube-dashboard
           else
-            git fetch origin "release-$RELEASE_VERSION"
-            git checkout "release-$RELEASE_VERSION"
+            git fetch origin "release/dashboard/$RELEASE_VERSION"
+            git checkout "release/dashboard/$RELEASE_VERSION"
             echo "Switched to release branch"
             cd ./scripts
             ./chart_releaser.sh --helm-chart-folder testkube-dashboard --branch true
           fi
 
         env:
+          SERVICE: dashboard
```

Comment:
Is the `SERVICE` environment variable used by `chart_releaser.sh`? Maybe it's worth to set the `SERVICE` environment variable for the whole workflow, and replace all `dashboard` segments with it?

Reply:
yes, it is by the chart_releaser.sh. 
Good point, will update with the var.